proposals/0216-widget-support.md

```diff
@@ -351,6 +384,18 @@ The `Show` request will take over the responsibilities of `SetDisplayLayout` by
 
 
 ```xml
+<struct name="TemplateConfiguration" since="5.x">
+  <param name="template" type="String" maxlength="500" mandatory="true">
+    <description>
+        Predefined or dynamically created window template.
+        Currently only predefined window template layouts are defined.
+    </description>
+  </param>
+
+  <param name="dayColorScheme" type="TemplateColorScheme" mandatory="false" />
```

Comment:
@kshala-ford Can you specify definition of type `TemplateColorScheme`. I can find it nowhere. Thank you